### ISSUE TITLE
Communications controller should have a bidirectional stream

### ISSUE THREAD

**Issue by menduz** [decentraland/sdk#582]
This API should use a bidirectional binary stream in sdk7, removing observables

**Comment by ProteanCoding** [comment]
I arrived here after finding the comment on https://github.com/decentraland/protocol/blob/main/proto/decentraland/kernel/apis/communications_controller.proto#L19.

There is no indication whatsoever as to how the binary stream is supposed to be used in the stead of the former text stream.
Now that the deprecated functionality is actually no longer working, some very critical scene tools are unavailable.
Notably the possibility of kicking disruptive users from a scene.

**TLDR: THE BOUNCER CODE NO LONGER WORKS**
